From 848096e16193b9eb60dfab92b60ce06d269b9147 Mon Sep 17 00:00:00 2001
From: Jerry Cao <jerry.cao@amlogic.com>
Date: Fri, 21 Mar 2014 00:42:42 -0700
Subject: [PATCH 3784/5965] [TrustZone] Add TrustZone kernel config

Commit 2b11fe4ac5cc85330891d220b3a1a0bff34856d1 removes MESON_TRUSTZONE
kenel config and thus breaks meson8 TrustZone enabled build.

Adding TrustZone config into meson8 Kconfig
---
 arch/arm/mach-meson8/Kconfig | 17 +++++++++++------
 1 file changed, 11 insertions(+), 6 deletions(-)

diff --git a/arch/arm/mach-meson8/Kconfig b/arch/arm/mach-meson8/Kconfig
index 4435c2199fe5..f02ad8ee0982 100755
--- a/arch/arm/mach-meson8/Kconfig
+++ b/arch/arm/mach-meson8/Kconfig
@@ -52,11 +52,16 @@ config MESON_FIX_SMP_SCALING_WARNING
 		Fix scaling warning which caused by smp_call_function
 
 config MESON_SUSPEND_FIRMWARE_BASE
-        hex
-        default 0x04f00000
-        depends on MESON_SUSPEND
-        help
-          The base address of 1MB of memory reserved for suspend firmware.
-endif # ARCH_MESON8
+	hex
+	default 0x04f00000
+	depends on MESON_SUSPEND
+	help
+		The base address of 1MB of memory reserved for suspend firmware.
 
+config MESON_TRUSTZONE
+	bool "Enable ARM Trustzone secure extension"
+	default n
+	help
+		Enable ARM Trustzone secure extension
 
+endif # ARCH_MESON8
-- 
2.19.0

